### PR TITLE
V3 route binding operation in progress

### PR DESCRIPTION
**Pull request by pivotal-marcela-campo**
[#174540070](https://www.pivotaltracker.com/story/show/174540070)

### CODE REVIEW

File: app/models/services/route_binding.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'cloud_controller/process_route_handler'
```
Comment (blgm) [inline app/models/services/route_binding.rb:1]: Does this relate to the change in spec helper?

File: app/repositories/app_usage_event_repository.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 require 'database/old_record_cleanup'
+require 'cloud_controller/url_secret_obfuscator'
```
Comment (blgm) [inline app/repositories/app_usage_event_repository.rb:2]: Does this relate to the change in spec helper?

**Review by blgm — APPROVED** [review]
There are a couple of new `requires`s that are not immediately obvious, but may relate to the change in spec helper. If they are intentional fixes that I'm fine with that, but just wanted to highlight them in case they are the result of the IDE doing strange things.

File: app/models/services/route_binding.rb (hunk 1 above)
Comment (pivotal-marcela-campo) [reply app/models/services/route_binding.rb:1]: yes!

File: app/repositories/app_usage_event_repository.rb (hunk 2 above)
Comment (pivotal-marcela-campo) [reply app/repositories/app_usage_event_repository.rb:2]: yes!

**Pull request merged**